firstvoices/backend/models/dictionary.py

```diff
@@ -173,10 +172,10 @@ class DictionaryEntry(AudienceMixin, RelatedMediaMixin, BaseControlledSiteConten
     #  (one character and one flag) used for sorting purposes. There is not much use of retaining sorting information
     #  after ~112 characters incase there are words which contain all 225 unknown characters. Thus, the field gets
     #  truncated at max length.
-    custom_order = TruncatingCharField(max_length=TITLE_MAX_LENGTH, blank=True)
+    custom_order = TruncatingCharField(max_length=DEFAULT_TITLE_LENGTH, blank=True)
 
     # from nxtag:tags
-    batch_id = models.CharField(max_length=255, blank=True)
+    batch_id = models.CharField(blank=True)
```

Comment:
Should this still have a max_length?

Reply:
Oops, I took this out at some point because the default length is 255 (so no actual change here), but I could add it back.